backend/expense/src/main/java/org/egov/digit/expense/repository/BillRepository.java

```diff
@@ -27,10 +27,10 @@ public BillRepository(JdbcTemplate jdbcTemplate, BillQueryBuilder queryBuilder,
 		this.searchBillRowMapper = searchBillRowMapper;
 	}
 
-	public List<Bill> search(BillSearchRequest billSearchRequest){
+	public List<Bill> search(BillSearchRequest billSearchRequest, boolean isValidationSearch){
```

Comment:
What is the meaning of this fields, not understandable by default. when will it be set to true and when false?

Reply:
added comment in repo layer where it is used